cli/internal/providers/datacatalog/empty_catalog.go

```diff
@@ -123,3 +123,19 @@ func (m *EmptyCatalog) DeleteCategory(ctx context.Context, categoryID string) er
 func (m *EmptyCatalog) GetCategory(ctx context.Context, id string) (*catalog.Category, error) {
 	return nil, nil
 }
+
+func (m *EmptyCatalog) GetEvents(ctx context.Context) ([]*catalog.Event, error) {
```

Comment:
I am assuming that `EmptyCatalog` struct is expected to implement the API client interface, that is why we added these methods. If yes, then shouldn't this code be in the api package and providers package?